woocommerce/woocommerce
# Fix Core Profiler layout shifts during plugin install step

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR fixes layout shifts during the plugin install step by adding a new `div` container inside the wrapper and applies new CSS so that the loader contents are all aligned to the top of the container and don't shift when the loader content changes. It also updates the SVG image assets so that they're all the same size (216 x 140px).

Closes #38762.



### How to test the changes in this Pull Request:




Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new WP site and install WooCommerce
2. Visit WooCommerce > Home in wp-admin to trigger the Core Profiler OR visit the URL directly at `{yoursite}/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard`
3. Step through the Core Profiler until you get to the plugin installer.
4. Uncheck all but the top two plugins (WooPayments and WooCommerce Shipping) so that the profiler has a chance to step through the loader.
5. The loader content should not shift down after the first loader change (see video below):

![2023-08-24 14 14 48](https://github.com/woocommerce/woocommerce/assets/4297811/319fc2d1-c145-4b61-9dd8-b470abe76d71)
